sdks/python/apache_beam/io/datastore/v1/datastoreio.py

```diff
@@ -285,3 +286,105 @@ def get_estimated_num_splits(project, namespace, query, datastore):
       num_splits = ReadFromDatastore._NUM_QUERY_SPLITS_MIN
 
     return max(num_splits, ReadFromDatastore._NUM_QUERY_SPLITS_MIN)
+
+
+class Mutate(PTransform):
```

Comment:
Should this class be marked as private (`_Mutate`). Would end users use it directly?